README.md

```diff
@@ -1,11 +1,54 @@
-# vanilla-monorepo
-Proof-of-concept / exploration of the Vanilla design system as a monorepo
+# DS-25 - Canonical's new design system
 
-## Installation
+Canonical's new design system.
+This is the successor to [Vanilla framework](https://vanillaframework.io).
 
-```sh   
-bun i
-git config diff.lockb.textconv bun
-git config diff.lockb.binary true
-```
+## Getting started
 
+1. Install [NodeJS](https://nodejs.org/en/download/package-manager) version 20 or later.
+2. Install the [bun package manager](https://bun.sh/).
+3. Configure your local git client to read Bun lockfiles as text: `git config diff.lockb.textconv bun && git config diff.lockb.binary true`.
```

Comment:
Is this needed to use it? Or is this some kind of "quality of life" developer option?

Reply:
It's not needed to use it but it does qualify as quality of life for sure. I'll move it to a separate section that is not in the main install notes.